frontend/web/styles/3rdParty/_hljs.scss

```
@@ -46,66 +54,73 @@
+      text-decoration: underline;
+      .hljs-regexp,
+      .hljs-comment {
+        color: var(--color-text-action) !important;
+      }
+    }
+    cursor: text;
+    outline: none;
+    font-size: 17px;
+    display: block;
+    overflow-x: auto;
+    @include customScrollDark();
```

Stylelint (17.14.0)

[error] Unexpected parentheses in argumentless mixin "customScrollDark" call (scss/at-mixin-argumentless-call-parentheses)

(scss/at-mixin-argumentless-call-parentheses)